src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -66,6 +68,8 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
 
   const { data, isLoading } = useCatalogueItems(parentInfo.id);
 
+  const { data: manufacturerList } = useManufacturers();
```

Comment:
It would be best to only load the unique manufactures - we may end up with a lot of them. This would probably involve using a typescript `Set` to obtain ids once the catalogue items data is loaded, then making use of react-query's `useQueries` method to obtain the results for these before matching them like you have already done.